prow/jobs/images/Dockerfile.integration

```diff
@@ -6,7 +6,8 @@ ARG DOCKER_VERSION=19.03.15
 ENV DOCKER_VERSION=${DOCKER_VERSION}
 
 RUN echo "Installing controller-gen v0.9.2..." \
-    && go install "sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2"
+    && go install "sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2" \
+    && mv $GOPATH/bin/controller-gen /usr/bin/controller-gen
```

Comment:
PR title says `/usr/local/bin` and this is to `/usr/bin`-- which one is correct?

Reply:
`/usr/bin/` ! changed :)